ehitchhiking-ui/src/app/pages/main-screen/yandex-map/yandex-map.component.sass

```diff
@@ -0,0 +1,6 @@
+#map
```

Comment:
pls remove extra styles for #map from project

Reply:
This style is needed here, in other sass files there is nothing about #map